### PR TITLE
Add camera python bindings

### PR DESCRIPTION
**Pull request by mwestphal**
Fix: https://github.com/f3d-app/f3d/issues/356

### CODE REVIEW

File: python/F3DPythonBindings.cxx [hunk 1]
```diff
@@ -103,6 +106,34 @@ PYBIND11_MODULE(f3d, module)
     .def(
       "getFileInfo", &f3d::loader::getFileInfo, "Get the file index, path, and information string");
 
+  // f3d::camera
+  py::class_<f3d::camera, std::unique_ptr<f3d::camera, py::nodelete> > camera(module, "camera");
+
+  camera.def("setPosition", &f3d::camera::setPosition)
+    .def("getPosition", py::overload_cast<>(&f3d::camera::getPosition))
+    .def("getPosition", py::overload_cast<f3d::point3_t&>(&f3d::camera::getPosition))
```
Comment (Meakk) [inline python/F3DPythonBindings.cxx:114]: Don't you have to bind the types too?

**Review by Meakk — CHANGES_REQUESTED** [review]
Please add a test

**Comment by mwestphal** [comment]
The python API is missing many tests. Should all bindings be tested ?

File: python/F3DPythonBindings.cxx (hunk 1 above)
Comment (mwestphal) [reply python/F3DPythonBindings.cxx:114]: Not sure what you mean, not a pybind11 expert here.

Comment (mwestphal) [reply python/F3DPythonBindings.cxx:114]: ok, got it now.

File: python/testing/TestPythonCamera.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import f3d
+import sys
```
Comment (Meakk) [inline python/testing/TestPythonCamera.py:2]: sys is not required I think

**Review by Meakk — APPROVED** [review]
Minor change but looks good to me

**Pull request merged**